app/Decsys/Repositories/LiteDb/LiteDbWordlistRepository.cs

```diff
@@ -54,7 +54,12 @@ public Task<WordlistWord> SetExcludedBuiltins(string wordlistId, string type, st
     {
         throw new NotImplementedException();
     }
-    
+
+    public Task<WordlistWord> AddCustomWord(string ownerId, string wordlistId, string type, string word)
+    {
+        throw new NotImplementedException();
```

Comment:
Is this intended for now?

Reply:
yes, liteDb is for the desktop app which doesn't support wordlist implementations. We've opted to go with only MongoDb for features that require an ownerId ( liteDb doesn't have users set up )

Reply:
Yes, wordlist does not work for liteDb which is used for the Desktop app. The Desktop app doesnt have users set up and any feature that relies on users such as wordlist only used MongoDb.